translate/src/cli.rs

```diff
@@ -33,7 +33,7 @@ pub struct Args {
 /// 1. Configurations passed using the `--config` command line flag.
 /// 2. A user-specific configuration directory (e.g. `$HOME/.config/harvest/config.toml').
 /// 3. Defaults specified in the code (using `#[serde(default)]`).
-#[derive(Debug, Deserialize)]
+#[derive(Default, Debug, Deserialize)]
```

Comment:
I don't think that auto-deriving `Default` is the right thing here, that will differ from the default config. I think that what `Default` should do is to deserialize `default_config.toml`.

In general, I would prefer a config system that uses `Default` as it's source of default values rather than `default_config.toml`, but that's now how the `config` crate works.